CONTRIBUTING.md

```diff
@@ -41,75 +25,97 @@ process documented here.
 
-### Before submitting a PR
+#### Use static checks
 
-Kata Containers utilizes a number of [CI systems](https://en.wikipedia.org/wiki/Continuous_integration)
-to automatically check every PR. As well as performing execution tests locally on
-your PR before submission, you are strongly encouraged to also run the same suite of
-[static checks](https://github.com/kata-containers/tests/blob/master/.ci/static-checks.sh)
-the CI will execute.
+* Kata Containers utilizes a number of [CI systems](https://en.wikipedia.org/wiki/Continuous_integration) to automatically check every PR. See [Continuous Integration](#continuous-integration).
 
-> **Note:** If working on `kata-runtime`, first ensure `make` and `make install` has been run in the `virtcontainers` subdirectory. (See [virtcontainers](https://github.com/kata-containers/runtime/blob/master/virtcontainers/documentation/Developers.md#testing) documentation for more information.)
-> 
-> ```sh
-> $ pushd runtime/virtcontainers
-> $ make
-> $ sudo -E PATH=$PATH make install
-> $ popd
-> ```
->
-> The final `popd` is required to return to the top-level directory from where other build rules can be executed.
+* We strongly encourage you to run the same CI tests on individual PRs, using [static checks](https://github.com/kata-containers/tests/blob/master/.ci/static-checks.sh)
 
-The easiest way to execute static checks in most of the repositories is to invoke the `Makefile` `check` and `test` rules, while developer mode is enabled.
+##### General
+
+To execute static checks in most repositories, invoke the `Makefile` `check` and `test` rules, while developer mode is enabled. 
 
 ```sh
 $ export KATA_DEV_MODE=true
 $ make check
 $ make test
 ```
-
 These should execute without errors. If errors are reported, fix them
 before submitting your PR.
 
-> **Note:** To ensure you replicate the static checks performed by the CI system, it is
-recommended that you:
-> * Ensure you have a "clean" source tree, as the checks will check all files present, and
-might fail if you have extra files or your files are out of date in your tree.
-> * Ensure that [`golangci-lint`](https://github.com/golangci/golangci-lint) is current or
-has not been installed (the static check scripts will install it if necessary). Changes
-in either the linters used or the Kata Containers code base can produce spurious errors
-that do not fail inside the CI systems.
+To replicate the static checks performed by the CI system:
+
+- [x] Ensure you have a "clean" source tree, as the checks will check all files present, and might fail if you have extra files or your files are out of date in your tree.
+
+- [x] Ensure that [`golangci-lint`](https://github.com/golangci/golangci-lint) is current or has not been installed (the static check scripts will install it if necessary). Changes in either the linters used or the Kata Containers code base can produce spurious errors that do not fail inside the CI systems.
+
+##### Kata runtime
```

Comment:
@jodh-intel - I think maybe the tests etc. will only work (at least for the runtime repo), if you are in the 'original' repo, and not a personal clone - because of the go path 'fun'... should we check/clarify that these instructions either mention that, or the recommendation due to use of golang is to work in the 'base' clone of the repo (not a personal one), but add a remote to your own personal fork where you push PRs iyswim?

**AH**, I see we cover that below here - maybe we need to ensure what is below (the fork and remote) matches what is above....